app/assets/stylesheets/welcome.scss

```diff
@@ -160,6 +160,29 @@ span.list-group-item.active {
   color: #e573fc;
   font-size: 10px;
 }
+
 .url-list {
   padding-bottom: 200px;
 }
+
+#flash_notice {
+  &.error {
+    border: solid 4px #900;
```

Comment:
Color literals like `#900` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered background-color, border